### PR TITLE
ticket 10

### PR DESCRIPTION
**Pull request by ellamarym**

### CODE REVIEW

File: src/components/ArticleList.jsx [hunk 1]
```diff
@@ -1,71 +1,111 @@
...
                 setIsLoading(false)
             })
-        } else {
+        } else{
            getAllArticles().then((articles) => {
             setArticlesList(articles)
             setIsLoading(false)
         }) 
         }
+        if(queries.length){
```
Comment (cjplatten) [inline src/components/ArticleList.jsx]: This if block and the previous if/else block are **both** running, meaning setArticlesList is being set twice, resulting in the articles not always being sorted.

**Review by cjplatten — COMMENTED** [review]
The sorting doesn't always work. Please see comments and submit another PR once you have made changes.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Ticket #11 - Deleting comments: Works as expected and is good to merge!

Ticket #10 - Sorting articles: Just need to refactor your axios requests into a single fetch using request config to control the optional queries 👍

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -1,86 +1,132 @@
...
+                <label htmlfor='chooseTopic'> Filter By Topic</label>
+                <select name='chooseTopic' onChange={topicHandler}>
+                    <option >Show all</option>
+                    {topicList.map(({ slug }) => {
+                        return <option value={slug}>{slug}</option>;
+                    })}
+                </select>
+                <label htmlFor="sortby">Sort by</label>
+                <select name='sortBy' onChange={sortHandler}>
+                    <option value='placeholder'></option>
```
Comment (cjplatten) [inline src/components/ArticleList.jsx:118]: the way that this is constructed means that if this placeholder is selected it sends a query to the api of "topic={topic}&placeholder"

**Review by cjplatten — COMMENTED** [review]
please see comment. 
There are also some warnings showing up in the console that it would be good to have a look at fixing

**Pull request merged**